pkg/certificate/cleanup_test.go

```diff
@@ -243,16 +233,15 @@ var _ = Describe("CABundle cleanup", func() {
 				},
 			},
 		}),
-		Entry("middle one beyond overlap duration right now, should remove it", cleanUpCertificatesCase{
-			overlapDuration: 5 * time.Hour,
+		Entry("middle one expired right now, should remove it", cleanUpCertificatesCase{
 			certsExpiration: []certificateExpiration{
 				{
 					notBefore: -1 * time.Hour,
 					notAfter:  33 * time.Hour,
 				},
 				{
 					notBefore: -5 * time.Hour,
-					notAfter:  44 * time.Hour,
+					notAfter:  -1 * time.Hour,
```

Comment:
shouldn't `expired right now` mean that the notAfter should be 0?

Reply:
Yep, fixing!!!